src/hearings/store/effects/hearing-request.effects.ts

```diff
@@ -61,32 +62,52 @@ export class HearingRequestEffects {
   public continueNavigation$ = this.actions$.pipe(
     ofType(hearingRequestActions.UPDATE_HEARING_REQUEST),
     tap(() => {
-      const nextPage = this.pageFlow.getNextPage(this.screenNavigations$);
+      let nextPage;
```

Comment:
The existing code should be fine? I think nextPage can remain as a const and its value can be fetched just above switch statement.

Reply:
the reason to change this to let is because some of the logic is changed for amend journey, so on VIEW_EDIT mode I don't want the this.pageFlow.getNextPage(this.screenNavigations$); to be called at first as it's not needed for amend journey. we can use the preview code but the problem is this is redudant call for VIEW_EDIT mode which I want to avoid, hope that make sense.